Marlin/ultralcd.cpp

```diff
@@ -836,7 +836,9 @@ void kill_screen(const char* lcd_msg) {
 
     void _lcd_user_gcode(const char * const cmd) {
       enqueue_and_echo_commands_P(cmd);
-      lcd_completion_feedback();
+      #ifdef USER_SCRIPT_FEEDBACK
```

Comment:
Yeah, I'm never sure if the preferred syntax is `#ifdef`, `#if ENABLED` or `#if defined`.  Happy to update the PR to whatever is most acceptable.  If I get a chance I'll poke around more to see what normally gets used.